crates/youki/src/commands/info.rs

```diff
@@ -26,10 +26,14 @@ pub fn info(_: Info) -> Result<()> {
     Ok(())
 }
 
-/// print Version of Youki
+/// print Version of Youki in Moby compatible format
```

Comment:
Don't remove the reference.
https://github.com/moby/moby/blob/65cc84abc522a564699bb171ca54ea1857256d10/daemon/info_unix.go#L280